CHANGELOG.md

```diff
@@ -7,6 +7,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 This changelog format was introduced in NAV 5.4.0. Older changelogs can be
 found in the [HISTORY](HISTORY) file.
 
+## [Unreleased]
+
+
+
+## bugfix/sysobjectid-with-dot
+
+### Fixed
+
+#### User-visible fixes
+- Validate and clean sysobjectid in NetboxTypeForm ([#2584](https://github.com/Uninett/nav/pull/2584), [#2566](https://github.com/Uninett/nav/issues/2566))
```

Comment:
Don't think end users will make much sense of what "NetboxTypeForm" is, so it should at least be mentioned that this fix is for SeedDB.
```suggestion
- Validate and clean sysobjectid input in SeedDB NetboxTypeForm ([#2584](https://github.com/Uninett/nav/pull/2584), [#2566](https://github.com/Uninett/nav/issues/2566))
```